.storybook/preview.js

```diff
@@ -34,6 +34,8 @@ addParameters({
   viewport: { viewports: newViewports },
 });
 
+// Hot Module Reloading (HMR)
+// force full reload to not reregister web components
```

Comment:
```suggestion
// force full reload not to reregister web components
```

Reply:
Wee keep "force full reload not to reregister web components" for searching easily